sda-svc/README.md

```diff
@@ -87,7 +87,7 @@ Parameter | Description | Default
 `global.elixir.pubKey` | Public key used to verify Elixir JWT. | `""`
 `global.elixir.jwkPath` | Path on the oicd Host where the JWK definitions can be found. | `/oidc/jwk`
 `global.elixir.oidcdHost` | URL to get the public key used to verify Elixir JWT. | `"https://login.elixir-czech.org"`
-`global.inbox.brokerRoutingKey` | Routing key the inbox uses when publishing messages. | `files.inbox"`
+`global.inbox.brokerRoutingKey` | Routing key the inbox uses when publishing messages. | `inbox"`
```

Comment:
should this be also in `values.yaml` ?

also adding `global.inbox.brokerExchange` with default to `sda` ?